### PR TITLE
Fixed issue #15057: Translation screen not working correctly when there are long question text

### PR DESCRIPTION
**Pull request by gabrieljenik**

### CODE REVIEW

File: application/controllers/admin/Translate.php
Comment (Shnoulle) [inline application/controllers/admin/Translate.php:219]: Maybe (for broken DB) check if parent exist ? 

`if (!empty($oRowfrom['parent_qid']) and !empty($oRowfrom->parent)) {`

I know it must be fixed by Check_data_integrity, then maybe leave it ?

File: application/controllers/admin/Translate.php
Comment (Shnoulle) [inline application/controllers/admin/Translate.php:937]: This part replace question text by quetion title ?

related to the issue fixed ?

**Comment by olleharstedt** [comment]
@Shnoulle You're waiting for feedback here before merge?

**Comment by Shnoulle** [comment]
Just a minute, this afternoon : it's OK ?

I just don't understand the update from question to title currently : i like to check (even if i don't do test)

**Comment by olleharstedt** [comment]
I'll wait for your OK.

**Review by Shnoulle — APPROVED** [review]

**Comment by Shnoulle** [comment]
Need to report the issue about Question text in place of question title (code)
And report for answer too :)

Else : commit is good :+1:

**Comment by Shnoulle** [comment]
I compare with 3.X : it's Question title .

But currently : there are same issue in answers, you report or i do it ?

**Comment by gabrieljenik** [comment]
> But currently : there are same issue in answers, you report or i do it ?

Please you do it, I'm not sure what you're referring to

**Pull request merged**
